Applications/TransferLearning/Draw_Classification/jni/Android.mk

```diff
@@ -62,7 +62,8 @@ LOCAL_LDLIBS := -llog
 
 LOCAL_SRC_FILES := main.cpp
 
-LOCAL_SHARED_LIBRARIES := capi-nntrainer app_utils
+# @todo add api for context
+LOCAL_SHARED_LIBRARIES := nntrainer capi-nntrainer app_utils
```

Comment:
capi-nntrainer depends on nntrainer. I dont think nntrainer needs to be specified here.
check line 38.

Reply:
Appcontext is not part of the `capi-nntrainer`, so without this line, the app throws DSO missing error

Reply:
Thats quite strange.